OlsztynHooligans/src/app/register/register.component.html

```diff
@@ -1,43 +1,84 @@
 <html>
+
 <body id="RegisterForm">
 
   <div class="container">
-    <h1 class="form-heading"style="font-family:Arial">Register</h1>
+    <h1 class="form-heading" style="font-family:Arial">Register</h1>
```

Comment:
w html nie robimy style.
chcesz to pod klase podpinasz. Lub dla wszystkich h1 etc robisz font-family